exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,278 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "For this exercise it is possible to implement concurrency tests",
+    "but for language tracks that do not support concurrency, the tests",
+    "are written to be run in a single thread.",
+    "But for language tracks that do support concurrency, so is it possible",
+    "to write tests that run concurrently."
+  ],
+  "cases": [
+    {
+      "uuid": "983a1528-4ceb-45e5-8257-8ce01aceb5ed",
+      "description": "Using pop raises an error if the list is empty",
```

Comment:
I don't recognise pop as an operation that https://github.com/exercism/problem-specifications/blob/main/exercises/bank-account/description.md should be made on a bank account, nor do I recall the description mentioning lists.

All descriptions in this file similarly reference terms that are unfamiliar to me when thinking about bank-account

Reply:
Thanks ✨